src/NuGet.Services.BasicSearch/ServiceEndpoints.cs

```diff
@@ -15,6 +15,7 @@ public class ServiceEndpoints
             var skip = GetSkip(context);
             var take = GetTake(context);
             var includePrerelease = GetIncludePrerelease(context);
+            var includeSemver2 = GetIncludeSemver2(context);
```

Comment:
My understanding of the result of the discussion was that semVerLevel >=2.0.0 would be treated as all inclusive, which in the current case is equivalent to semVerLevel==2.0.0.

Was this not the case?